lib/function.gi

```diff
@@ -59,3 +59,26 @@ function(func)
     Append(result, " ) ... end");
     return result;
 end);
+
+
+InstallMethod(DisplayString, "for a function, using string stream", [IsFunction], 
+function(fun)
+    local  s, str;
+    s := "";
+    str := OutputTextString(s, true);
+    PrintTo(str, fun);
+    CloseStream(str);
```

Comment:
Super-nitpick, I am not expecting any action on it, but needed to get it off my chest: When I see "str" I think "string", but here it is "stream", while "s" is a string... This function is short, and so it's harmless, but I'd still rename at least one of them (probably `str` to `stream`, as I see strings more more often than streams, at least in GAP code)